t/fixtures/01-jobs.pl

```diff
@@ -23,7 +23,7 @@
     Assets => {
         id   => 5,
         type => 'hdd',
-        name => 'openSUSE-13.1-x86_64.hda.xz',
```

Comment:
this was just bogus, as the compressed file would never actually have become a job asset. it would only have been registered as an asset at all by the big asset registration hammer in `limit_assets`.